internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -378,16 +378,16 @@ func (w *sessionWorkflow) getNextEvent(ctx context.Context, signalID string) (ma
 	iid := sdktypes.NewIDFromUUID[sdktypes.IntegrationID](signal.Connection.IntegrationID)
 	cid := sdktypes.NewIDFromUUID[sdktypes.ConnectionID](&signal.Connection.ConnectionID)
 
-	minSequenceNumber, ok := w.signals[signalID]
+	minSequenceNumber, ok := w.lastReadEventSeqForSignal[signalID]
 	if !ok {
-		w.z.Panic("signal not found")
+		return nil, fmt.Errorf("no such subscription %q", signalID)
 	}
 
 	filter := sdkservices.ListEventsFilter{
 		IntegrationID:     iid,
 		ConnectionID:      cid,
 		Limit:             1,
-		MinSequenceNumber: minSequenceNumber,
+		MinSequenceNumber: minSequenceNumber + 1,
```

Comment:
I'm not sure why +1 here and >= in the db layer makes a difference 

it's suppose to be the same as minSequenceNumber and >

Reply:
if it's min AND >, it'll not include the min. I want it to include the min.